readme.txt

```diff
@@ -35,11 +35,13 @@ To install the External Videos plugin simply:
 
 = What sites are supported? =
 
-Currently supported sites are: YouTube, Vimeo, DotSub
+Currently supported sites are: YouTube, Vimeo
```

Comment:
the media gallery hooks don't work anymore because of media using backbone.js, which i have to learn. for next release!